.github/ISSUE_TEMPLATE/contributions.yml

```
@@ -0,0 +1,30 @@
+name: 👷 Contribute to next priorities
+description: Help us with our work on our next priorities 
+title: 'contribution: '
+body:
+  - type: checkboxes
+    attributes:
+      label: Prerequisites
+      description: Complete all of the following.
+      options:
+        - label: I've read the [Contributing Guidelines](https://github.com/cds-snc/gcds-components/blob/main/CONTRIBUTING.md).
+          required: true
+        - label: I agree to follow the [Code of Conduct](https://github.com/cds-snc/gcds-components/blob/main/CODE_OF_CONDUCT.md).
+          required: true
+        - label: I've searched for [existing issues](https://github.com/cds-snc/gcds-components/issues) that already report this problem, without success.
```

Should we replace "this problem" with something else? If we are talking about "next priorities", it might be more focused on new features.

---

@melaniebmn 
```suggestion
        - label: I've searched for [existing issues](https://github.com/cds-snc/gcds-components/issues) that already tackle this specific contribution, without success.
```

Agree -- how about this for a first pass? Also tagged Brock and Amy here so it may need further revisions as well

---

Looks good to me.  I'd even be in favor of the person submitting not having to verify existing for "next priorities" contribution.

---

I can remove that checkbox to lesson the load on the user.